### PR TITLE
Clean up show/hide/and add get visibility method on Windows

### PR DESCRIPTION
**Pull request by musm**

### CODE REVIEW

**Comment by musm** [comment]
thoughts?

**Comment by musm** [comment]
bump

**Comment by musm** [comment]
bump

File: src/MATLAB.jl [hunk 1]
```diff
@@ -31,9 +31,11 @@ export MSession, MatFile,
        eval_string, get_mvariable, get_variable, put_variable, put_variables,
        variable_names, read_matfile, write_matfile,
        mxcall,
-       @mput, @mget, @matlab, @mat_str,
-       show_msession, hide_msession
+       @mput, @mget, @matlab, @mat_str
 
+if is_windows()
+    export show_msession, hide_msession, get_msession_visiblity
```
Comment (yuyichao) [inline src/MATLAB.jl]: Do these only exist on Windows?

Comment (musm) [inline src/MATLAB.jl]: yeah 
ref:
https://www.mathworks.com/help/matlab/apiref/engsetvisible.html
https://www.mathworks.com/help/matlab/apiref/enggetvisible.html

**Comment by musm** [comment]
thanks!

**Comment by yuyichao** [comment]
I assume no one else has more comment that are not posted in the past 3weeks...

**Pull request merged**